src/portal/src/app/base/harbor-shell/harbor-shell.component.ts

```diff
@@ -82,8 +88,29 @@ export class HarborShellComponent implements OnInit, OnDestroy {
         this.searchCloseSub = this.searchTrigger.searchCloseChan$.subscribe(close => {
             this.isSearchResultsOpened = false;
         });
+        if (!(localStorage && localStorage.getItem(HAS_SHOWED_SCANNER_INFO) === 'yes')) {
+           this.getDefaultScanner();
+        }
+    }
+    closeInfo() {
+        if (localStorage) {
+            localStorage.setItem(HAS_SHOWED_SCANNER_INFO, 'yes');
+        }
+        this.showScannerInfo = false;
     }
 
+    getDefaultScanner() {
+        this.scannerService.getScanners()
+            .subscribe(scanners => {
+                if (scanners && scanners.length) {
```

Comment:
this.showScannerInfo = scanners.some(scanner =>scanner.is_default);   
One line of code can do it

Reply:
fixed